.nais/naiserator-dev.yaml

```diff
@@ -14,16 +14,16 @@ spec:
     cpuThresholdPercentage: 70
   liveness:
     path: /is_alive
-    initialDelay: 10
+    initialDelay: 60
     periodSeconds: 5
     timeout: 5
-    failureThreshold: 30
+    failureThreshold: 3
   readiness:
-    path: /is_alive
-    initialDelay: 30
+    path: /is_ready
+    initialDelay: 60
     periodSeconds: 5
     timeout: 5
-    failureThreshold: 30
+    failureThreshold: 3
```

Comment:
failureThreshold er *antall ganger* proben må feile etter hverandre for at Kubernetes skal restarte pod'en (for liveness) eller slutte å rute trafikk til pod'en (for readiness). Default er 3 (og 30 er alt for mye).